### PR TITLE
[llama] Remove unnecessary model attribute assignment on 'freqs_cis'

### LINKED ISSUE #1767: [export][llama] AssertionError: Mutating module attribute freqs_cis during export
```
        Mutating module attribute freqs_cis during export.
        
        from user code:
           File "/bert_ort/bowbao/repos/bench/torchbenchmark/torchbenchmark/models/llama/model.py", line 227, in forward
            self.freqs_cis = self.freqs_cis.to(h.device)
        
        Set TORCH_LOGS="+dynamo" and TORCHDYNAMO_VERBOSE=1 for more information
        
        
        You can suppress this exception and fall back to eager by setting:
            import torch._dynamo
            torch._dynamo.config.suppress_errors = True       
```

@msaroufim What's the general recommendation for this kind of error? Feels like the model code should be altered?

### PR DESCRIPTION
**Pull request by BowenBao**
Fixes #1767, more details there.

Upstream PR link https://github.com/facebookresearch/llama/pull/349

### CODE REVIEW

**Review by msaroufim — APPROVED** [review]

**Comment by msaroufim** [comment]
CI failure seems to be a flake not your fault, id retry triggering build

**Comment by xuzhao9** [comment]
The upstream model code has a similar issue: https://github.com/facebookresearch/llama/blob/main/llama/model.py#L226

The convention is before we merge this PR, we require the PR author to create a PR to fix the upstream model code as well, and reference the upstream PR link in the code of torchbench model.

File: torchbenchmark/models/llama/model.py [hunk 1]
```diff
@@ -224,8 +224,8 @@ def forward(self, tokens: torch.Tensor, start_pos: int):
    
         h = self.tok_embeddings(tokens)
 
-        self.freqs_cis = self.freqs_cis.to(h.device)
-        freqs_cis = self.freqs_cis[start_pos : start_pos + seqlen]
+        freqs_cis = self.freqs_cis.to(h.device)
```
Comment (xuzhao9) [inline torchbenchmark/models/llama/model.py:227]: Can we add a reference to https://github.com/facebookresearch/llama/pull/349 here?

Comment (BowenBao) [inline torchbenchmark/models/llama/model.py:227]:
```suggestion
        # Reference: https://github.com/facebookresearch/llama/pull/349
        freqs_cis = self.freqs_cis.to(h.device)
```

Comment (BowenBao) [reply torchbenchmark/models/llama/model.py:228]: Done

**Review by xuzhao9 — APPROVED** [review]
LGTM!

**Pull request closed without merging**